printer_mods/chirpy/NozzleWiper/nozzlewiper.cfg

```diff
@@ -0,0 +1,99 @@
+###############################################################################
+## NOZZLE WIPER - A nozzle purge bucket and brush for the Voron 0.1 and other
+##                printers for ants
+##
+## Configuration:
+##
+## 1.  Set the correct PMW pin that you connected the Orange wire to in the
+##     [servo wipeServo] section
+## 2.  Home X, Y and Z axes
+## 3.  Lower Z by at least 50mm
+## 4.  Run the NW_DEPLOY macro and ensure the servo extends the arm 90 degrees
+## 5.  Use the printer controls to move the nozzle so that it is inline with
+##     the brush on the X axis and so that it is in the centre of the purge
+##     bucket on the Y axis
+## 7.  Run M114 to get the nozzle position and set variable_x and variable_y
+## 8.  Move the nozzle into the center of the brush and raise the mount on
+##     the extrusion until the nozzle just rests into the brush. Do not let the
+##     brush touch the sock or it'll wear it out prematurely
+## 9.  Move the nozzle out of the way from the brush and purge bucket
+## 10. Run NW_RETRACT and ensure the servo moves the arm back away to the side
+##
+## Testing:
+##
+## 1.  Heat the nozzle to the printing temperature of the loaded filament
+## 2.  Home all axes
+## 3.  Be prepared to hit the Emergency Stop button if things don't go as they
+##     should
+## 4.  Run NW_CLEAN_NOZZLE
+##
+## That should be all changes needed. Call the NW_CLEAN_NOZZLE macro from
+## the end of your START_PRINT macro to wipe the nozzle clean before printing
+###############################################################################
+
+[servo wipeServo]
+pin: PA8
```

Comment:
Can you add a comment for which board this pin definition is for? I would also consider commenting out the pin definition entirely to force the user to research what the correct pin is for their board.

Reply:
I will set the pin to a value that they will need to change.